src/app/management-center/catalogs/catalogs.component.html

```diff
@@ -40,9 +40,7 @@
             <button class="action-item" (click)="goToDetail(catalog)">Details</button>
         </clr-dg-action-overflow>
       </clr-dg-row>
-      <clr-dg-footer>
-        <inventory-footer></inventory-footer>
-      </clr-dg-footer>
+      <clr-dg-footer></clr-dg-footer>
```

Comment:
Not sure if possible but if you are not putting anything inside of `<clr-dg-footer></clr-dg-footer>` so can't it be also removed ?

Reply:
The problem is that if Clarity need the footer to render the [show/hide columns component](https://clarity.design/documentation/datagrid/hide-show). So you need the `</clr-dg-footer>` even if its empty, because clarity will add/render this component:

![Screenshot 2019-12-17 at 09 58 52](https://user-images.githubusercontent.com/44772295/70985329-dc91d100-20b3-11ea-9050-8c5702074f78.png)

Reply:
Oh i see..